libs/compatibility-spy/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@winglang/compatibility-spy",
```

Comment:
Will need to add this to the packages for publishing in github workflow

https://github.com/winglang/wing/blob/main/.github/workflows/build.yml#L388

Reply:
Also if we do any hangar testing with this platform will need to update `tools/hangar/src/package.setup.ts` and `tools/hangar/package.json`